### ISSUE TITLE
Clarify that string interpolation uses the *invariant* culture

### ISSUE THREAD

**Issue by mklement0** [MicrosoftDocs/PowerShell-Docs#10953]
### Type of issue

Missing information

### Feedback

By design - so as to provide stability across different cultures - PowerShell's string interpolation uses the [_invariant_ culture](https://learn.microsoft.com/en-US/dotnet/api/System.Globalization.CultureInfo.InvariantCulture), which is worth mentioning explicitly.

For instance, with the `en-US` culture in effect:

`(Get-Date 1970-01-01).ToString()` yields a culture-specific representation, `1/1/1970 12:00:00 AM`

By contrast, `"$(Get-Date 1970-01-01)"` yields the following representation, `01/01/1970 00:00:00`, _irrespective of what culture is in effect_.

In other words: When PowerShell performs string interpolation on non-string values, instead of calling `.ToString()`,
it calls `.ToString([cultureinfo]::InvariantCulture)` (where supported by the type at hand).

### Page URL

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_Quoting_Rules?view=powershell-7.4

### Content source URL

https://github.com/MicrosoftDocs/PowerShell-Docs/blob/main/reference/7.4/Microsoft.PowerShell.Core/About/about_Quoting_Rules.md

### Author

@sdwheeler

### Document Id

0b3f8e35-a9eb-affd-4bef-c6d584373914
